var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -169,6 +170,22 @@ def install(self, spec, prefix):
         except OSError:
             pass
 
+        # Environment views do not currently support symlinks.
+        # Since the include/ and lib64/ directories in cuda are
+        # symlinks to subdirectories of targets/ and thus do
+        # not appear in the view, we manually replace them with
+        # copies of the directories they point to.
+        # For reference, see https://github.com/spack/spack/issues/19531
+        include_dir = join_path(prefix, 'include')
+        pointing_to = join_path(prefix, os.readlink(include_dir))
+        os.remove(include_dir)
+        shutil.copytree(pointing_to, include_dir)
+
+        lib64_dir = join_path(prefix, 'lib64')
+        pointing_to = join_path(prefix, os.readlink(lib64_dir))
+        os.remove(lib64_dir)
+        shutil.copytree(pointing_to, lib64_dir)
```

Comment:
Can you check if we have a substantial increase in size of the installed binaries?

Reply:
I think it will. On my system, the directory we want to copy is 3GB